--- a/configure
+++ b/configure
@@ -6733,6 +6733,7 @@
     test "x$enable_xml"      = "xyes" && pl="$pl libroot-io-xmlparser"
     test "x$enable_mathmore" = "xyes" && pl="$pl libroot-math-mathmore"
     test "x$enable_unuran"   = "xyes" && pl="$pl libroot-math-unuran"
+    test "x$enable_memstat"  = "xyes" && pl="$pl libroot-misc-memstat"
     test "x$enable_vc"       = "xyes" && pl="$pl libroot-math-vc"
     test "x$enable_vdt"      = "xyes" && pl="$pl libroot-math-vdt"
     test "x$enable_table"    = "xyes" && pl="$pl libroot-misc-table"
