--- a/cmake/scripts/testsingle.cxx.in
+++ b/cmake/scripts/testsingle.cxx.in
@@ -635,7 +635,7 @@
      filename = &filename1;
      */
    } else {
-     fprintf(stderr,"Wrong comand line. Please use\n");
+     fprintf(stderr,"Wrong command line. Please use\n");
      fprintf(stderr,"%s -f <filename> \n",argv[0]);
      return 1;     
    }
--- a/core/rint/inc/TTabCom.h
+++ b/core/rint/inc/TTabCom.h
@@ -232,7 +232,7 @@
    Bool_t fVarIsPointer;                         // frodo: pointer or not flag
    Int_t  fLastIter;                             // frodo: iteration counter for recursive MakeClassFromVarName
 
-   ClassDef(TTabCom,0)  //Perform comand line completion when hitting <TAB>
+   ClassDef(TTabCom,0)  //Perform command line completion when hitting <TAB>
 };
 
 R__EXTERN TTabCom *gTabCom;
--- a/gui/sessionviewer/src/TSessionViewer.cxx
+++ b/gui/sessionviewer/src/TSessionViewer.cxx
@@ -932,7 +932,7 @@
    tf->AddFrame(fFC, new TGLayoutHints(kLHintsTop | kLHintsLeft |
          kLHintsExpandX | kLHintsExpandY));
 
-   // add comand line label and text entry
+   // add command line label and text entry
    TGCompositeFrame* frmCmd = new TGHorizontalFrame(fFC, 350, 100);
    frmCmd->SetCleanup(kDeepCleanup);
    frmCmd->AddFrame(new TGLabel(frmCmd, "Command Line :"),
--- a/tutorials/gui/guiWithCINT.C
+++ b/tutorials/gui/guiWithCINT.C
@@ -1,7 +1,7 @@
 // A simple example of entering  CINT commands and having the CINT output in a 
 // ROOT GUI application window. 
 // An editable combo box is used as a CINT prompt, a text view widget displays 
-// the comand output.
+// the command output.
 // Author: Ilka Antcheva   06/07/2007
 
 #include <iostream>
