--- a/documentation/primer/macros/macro1.C
+++ b/documentation/primer/macros/macro1.C
@@ -22,7 +22,7 @@
 
     // Instance of the graph
     TGraphErrors graph(n_points,x_vals,y_vals,NULL,y_errs);
-    graph.SetTitle("Measurement XYZ;lenght [cm];Arb.Units");
+    graph.SetTitle("Measurement XYZ;length [cm];Arb.Units");
 
     // Make the plot estetically better
     graph.SetMarkerStyle(kOpenCircle);
--- a/documentation/primer/macros/macro2.C
+++ b/documentation/primer/macros/macro2.C
@@ -8,7 +8,7 @@
                                 "%lg %lg %lg");
     graph_expected.SetTitle(
        "Measurement XYZ and Expectation;
-        lenght [cm];
+        length [cm];
         Arb.Units");
     graph_expected.SetFillColor(kYellow);
     graph_expected.DrawClone("E3AL"); // E3 draws the band
--- a/graf2d/cocoa/src/TGCocoa.mm
+++ b/graf2d/cocoa/src/TGCocoa.mm
@@ -2836,7 +2836,7 @@
 //______________________________________________________________________________
 Int_t TGCocoa::TextWidth(FontStruct_t font, const char *s, Int_t len)
 {
-   // Return lenght of the string "s" in pixels. Size depends on font.
+   // Return length of the string "s" in pixels. Size depends on font.
    return fPimpl->fFontManager.GetTextWidth(font, s, len);
 }
 
