diff --git a/roofit/roofit/inc/Roo2DMomentMorphFunction.h b/roofit/roofit/inc/Roo2DMomentMorphFunction.h
index ca70236..99ec216 100644
--- a/roofit/roofit/inc/Roo2DMomentMorphFunction.h
+++ b/roofit/roofit/inc/Roo2DMomentMorphFunction.h
@@ -7,7 +7,7 @@
 // 2-dimensional morph function between a list of input functions (varlist) as a function of one input parameter (m).
 // The vector mrefpoints assigns an m-number to each function in the function list.
 // For example: varlist can contain MC histograms (or single numbers) of a reconstructed mass, for certain 
-// true Higgs masses indicated in mrefpoints. the input parameter m is the true (continous) Higgs mass.
+// true Higgs masses indicated in mrefpoints. the input parameter m is the true (continuous) Higgs mass.
 // Morphing can be set to be linear or non-linear, or mixture of the two.
 
 #ifndef ROO2DMOMENTMORPHFUNCTION
diff --git a/roofit/roofit/src/Roo1DMomentMorphFunction.cxx b/roofit/roofit/src/Roo1DMomentMorphFunction.cxx
index ab80fd4..79536f1 100644
--- a/roofit/roofit/src/Roo1DMomentMorphFunction.cxx
+++ b/roofit/roofit/src/Roo1DMomentMorphFunction.cxx
@@ -7,7 +7,7 @@
 // 1-dimensional morph function between a list of input functions (varlist) as a function of one input parameter (m).
 // The vector mrefpoints assigns an m-number to each function in the function list.
 // For example: varlist can contain MC histograms (or single numbers) of a reconstructed mass, for certain 
-// true Higgs masses indicated in mrefpoints. the input parameter m is the true (continous) Higgs mass.
+// true Higgs masses indicated in mrefpoints. the input parameter m is the true (continuous) Higgs mass.
 // Morphing can be set to be linear or non-linear, or mixture of the two.
 
 #include "Riostream.h" 
