Generate iosenum.h if not officially provided for the arch.

--- a/cint/cint/Module.mk
+++ b/cint/cint/Module.mk
@@ -301,15 +301,17 @@
 $(IOSENUM): 
 		$(MAKEDIR)
 		@(if [ ! -r $(IOSENUMA) ]; then \
-			echo "Missing $(IOSENUMA), run: make $IOSENUMA)"; \
-		else \
-			cp $(IOSENUMA) $@; \
-		fi)
+			echo "Missing $(IOSENUMA), run: make $(IOSENUMA)"; \
+			$(MAKE) $(IOSENUMA); \
+		fi; \
+		cp $(IOSENUMA) $@)
 
 $(IOSENUMA):    $(CINTTMP)
 		@(if [ ! -r $@ ]; then \
 			echo "Making $@..."; \
-			$(CINTTMP) $(CINTTMPINC) -Iinclude $(IOSENUMC) > /dev/null; \
+			$(CINTTMP) $(CINTTMPINC) \
+			        -I$(ROOT_SRCDIR)/cmake/scripts -Iinclude \
+				$(IOSENUMC) > /dev/null; \
 			mv iosenum.h $@; \
 		else \
 			touch $@; \
