--- a/bindings/ruby/Module.mk
+++ b/bindings/ruby/Module.mk
@@ -87,5 +87,5 @@
 distclean::     distclean-$(MODNAME)
 
 ##### extra rules ######
-$(RUBYROOTO): CXXFLAGS += $(RUBYINCDIR:%=-I%) -Iinclude/cint
-$(RUBY64O): CFLAGS += $(RUBYINCDIR:%=-I%)
+$(RUBYROOTO): CXXFLAGS += $(RUBYINCDIR:%=-I%) -I$(RUBYARCHINC) -Iinclude/cint
+$(RUBY64O): CFLAGS += $(RUBYINCDIR:%=-I%) -I$(RUBYARCHINC)
--- a/configure
+++ b/configure
@@ -4300,7 +4300,7 @@
        test "x$enable_shared" = "xno" ; then
         check_library_options="libruby-static libruby1.8-static libruby1.9-static"
     else
-        check_library_options="libruby libruby1.8 libruby18 libruby1.9 libruby19"
+        check_library_options="libruby-1.9.1 libruby1.9 libruby19 libruby1.8 libruby18 libruby"
     fi
     check_library "$check_library_options" \
         "$enable_shared" "$rubylibdir" \
