Index: b/configure
===================================================================
--- a/configure
+++ b/configure
@@ -7075,7 +7075,7 @@ rm -f lib/libRXML* bin/libRXML* etc/syst
 # First win32 does not support -L in ldflags, so remove @ldflags@.
 #
 if test "x$platform" = "xwin32" ; then
-    sed -e "s|@ldflags@||"        \
+    sed -e "s|@ldflags@|$EXTRA_LDFLAGS|"        \
         < Makefile.tmp > Makefile.tmpp
     mv Makefile.tmpp Makefile.tmp
 fi
@@ -7346,7 +7346,7 @@ sed -e "s|@globusincdir@|$globusincdir|"
     -e "s|@ldapincdir@|$ldapincdir|"            \
     -e "s|@ldaplib@|$ldaplib|"                  \
     -e "s|@ldaplibdir@|$ldaplibdir|"            \
-    -e "s|@ldflags@||"                          \
+    -e "s|@ldflags@|$EXTRA_LDFLAGS|"            \
     -e "s|@libdir@|$libdir|"                    \
     -e "s|@llvmconfig@|$llvmconfig|"            \
     -e "s|@macosxvers@|$macosxvers|"            \
