--- a/graf2d/asimage/src/TASImage.cxx
+++ b/graf2d/asimage/src/TASImage.cxx
@@ -2619,9 +2619,9 @@
    //    dissipate      - randomly alpha-blend images
    //    hue            - hue bottom image same as top image
    //    lighten        - use highest color value from both images
-   //    overlay        - some wierd image overlaying(see GIMP)
+   //    overlay        - some weird image overlaying(see GIMP)
    //    saturate       - saturate bottom image same as top image
-   //    screen         - another wierd image overlaying(see GIMP)
+   //    screen         - another weird image overlaying(see GIMP)
    //    sub            - color substraction with saturation
    //    tint           - tinting image with image
    //    value          - value bottom image same as top image
--- a/graf2d/asimage/src/libAfterImage/ChangeLog
+++ b/graf2d/asimage/src/libAfterImage/ChangeLog
@@ -510,7 +510,7 @@
 	  up wharf's support for look defined folder icon. That should also
 	  speedup repeated changes of look
 	* adding support in Wharf for look-specific folder pixmap - so far
-	  only works 2 times, then stops changing - wierd
+	  only works 2 times, then stops changing - weird
 
 2007-07-19  sasha
 
@@ -1059,7 +1059,7 @@
 
 2005-04-22  sasha
 
-	* Fixed off-by-one bug in ASStorage causing wierd artefact and
+	* Fixed off-by-one bug in ASStorage causing weird artefact and
 	  possibly segfaults, due to occupied slots being reused improperly
 
 2005-04-21  sasha
@@ -1381,7 +1381,7 @@
 	* fixed segfault in asstorage when diff buffer does not gets
 	  reallocated
 	* fixed button ungrabbing on windows withdrawing that was causing
-	  wierd BadAccess errors in Wharf
+	  weird BadAccess errors in Wharf
 
 2004-09-23  sasha
 
@@ -1527,7 +1527,7 @@
 	  calls to storage destruction code
 	* fixed thresholding code in asstorage needed for rectangular shape
 	  stuff
-	* Improved reference counting in ASStorage to accomodate to wierd
+	* Improved reference counting in ASStorage to accomodate to weird
 	  storage states where some blocks have no space left for reference
 
 2004-07-30  sasha
@@ -1738,7 +1738,7 @@
 
 2004-03-18  sasha
 
-	* Found and fixed very wierd heap corruption bug, causing segfault
+	* Found and fixed very weird heap corruption bug, causing segfault
 	  whenever bevelled rectang is drawn
 	* Added builtin libungif to libAfterImage, and enabled support for
 	  it by default in configure
@@ -1839,7 +1839,7 @@
 2004-01-21  sasha
 
 	* fixed bug where empty locale would be specified while starting a
-	  module causing Pager to have 255 desktops and other wierdness
+	  module causing Pager to have 255 desktops and other weirdness
 
 2004-01-13  sasha
 
@@ -1917,7 +1917,7 @@
 
 	* Added ActiveDeskBevel and InActiveDeskBevel to Pager's options;
 	  Debugged bevel drawing when tbar has different bevels for focus
-	  and unfocus states( should get rid of wierd black frame when one
+	  and unfocus states( should get rid of weird black frame when one
 	  of the states is None)
 
 2003-09-25  sasha
@@ -2121,7 +2121,7 @@
 	  TakeFrameShot, as the side effect of debugging root background
 	  handling. Updated feel.DEFAULT to use those
 	* fixed bug in inheritance code where back_icon would get destroyed
-	  even if it was inherited, thus causing wierd things to happen
+	  even if it was inherited, thus causing weird things to happen
 	  when same style is inherited twice; Added code to
 	  mystyle_make_image to ensure that we always produce valid ASImage
 
@@ -3646,7 +3646,7 @@
 	*  minor quality tweaks in rations 1-2 scaling up. Started
 	  implementing ASImageDecoder for automatic tinting/tiling
 	* IMplemented JPEG input filter for ASImage.  Fixed long standing
-	  bug in image scaling causing wierdness in scaling UP with small
+	  bug in image scaling causing weirdness in scaling UP with small
 	  ratios.  Started implemntation of XCF file reader.
 
 2001-04-15  sasha
--- a/graf2d/asimage/src/libAfterImage/afterbase.c
+++ b/graf2d/asimage/src/libAfterImage/afterbase.c
@@ -88,7 +88,7 @@
 	do
 	{	/* Save our program name - for error messages */
 		register int i = 1 ;                   /* we don't use standard strrchr since there
-												* seems to be some wierdness in
+												* seems to be some weirdness in
 												* CYGWIN implementation of it. */
 		asim_ApplicationName =  temp ;
 		while( temp[i] && temp[i] != '/' ) ++i ;
--- a/graf2d/asimage/src/libAfterImage/blender.c
+++ b/graf2d/asimage/src/libAfterImage/blender.c
@@ -319,9 +319,9 @@
   { "dissipate", 9, dissipate_scanlines, "randomly alpha-blend images"},
   { "hue", 3, hue_scanlines, "hue bottom image same as top image"  },
   { "lighten", 7, lighten_scanlines, "use highest color value from both images" },
-  { "overlay", 7, overlay_scanlines, "some wierd image overlaying(see GIMP)" },
+  { "overlay", 7, overlay_scanlines, "some weird image overlaying(see GIMP)" },
   { "saturate", 8, saturate_scanlines, "saturate bottom image same as top image"},
-  { "screen", 6, screen_scanlines, "another wierd image overlaying(see GIMP)" },
+  { "screen", 6, screen_scanlines, "another weird image overlaying(see GIMP)" },
   { "sub", 3, sub_scanlines, "color substraction with saturation" },
   { "tint", 4, tint_scanlines, "tinting image with image" },
   { "value", 5, value_scanlines, "value bottom image same as top image" },
--- a/graf2d/asimage/src/libAfterImage/blender.h
+++ b/graf2d/asimage/src/libAfterImage/blender.h
@@ -197,9 +197,9 @@
  *                        value of top and bottom :
  *                        bottom = (bottom>top)?bottom:top;
  * NAME
- * screen_scanlines()   - some wierd merging algorithm taken from GIMP;
+ * screen_scanlines()   - some weird merging algorithm taken from GIMP;
  * NAME
- * overlay_scanlines()  - some wierd merging algorithm taken from GIMP;
+ * overlay_scanlines()  - some weird merging algorithm taken from GIMP;
  * NAME
  * hue_scanlines()      - substitute hue of bottom scanline with hue of
  *                        top scanline;
--- a/graf2d/asimage/src/libAfterImage/transform.c
+++ b/graf2d/asimage/src/libAfterImage/transform.c
@@ -2639,7 +2639,7 @@
 		gauss[0] = 1.0;
 		return;
 	}
-	/* after radius of 128 - gaussian degrades into something wierd, 
+	/* after radius of 128 - gaussian degrades into something weird, 
 	   since our colors are only 8 bit */
 	if (radius > 128.0) radius = 128.0; 
 	std_dev = (radius - 1) * 0.3003866304;
@@ -2707,7 +2707,7 @@
 		gauss_sums[0] = 1024;
 		return;
 	}
-	/* after radius of 128 - gaussian degrades into something wierd, 
+	/* after radius of 128 - gaussian degrades into something weird, 
 	   since our colors are only 8 bit */
 	if (i > 128) i = 128; 
 #if 1
