--- a/proof/proof/src/TProofMgr.cxx
+++ b/proof/proof/src/TProofMgr.cxx
@@ -1031,7 +1031,7 @@
 void TProofDesc::Print(Option_t *) const
 {
    // Dump the content to the screen.
-   const char *st[] = { "unknown", "idle", "processsing", "shutting down"};
+   const char *st[] = { "unknown", "idle", "processing", "shutting down"};
 
    Printf("// # %d", fLocalId);
    Printf("// alias: %s, url: \"%s\"", GetTitle(), GetUrl());
