diff --git a/tree/tree/src/TEntryListArray.cxx b/tree/tree/src/TEntryListArray.cxx
index 532d74e..c5df1df 100644
--- a/tree/tree/src/TEntryListArray.cxx
+++ b/tree/tree/src/TEntryListArray.cxx
@@ -157,7 +157,7 @@ void TEntryListArray::Add(const TEntryList *elist)
 
    // Include in this list all the trees present in elist, so the sublists can be added
    // This would happen in any case when calling TEntryList::Add
-   if (elist->GetLists()) { // the other list has lists to hold mutiple trees, add one by one
+   if (elist->GetLists()) { // the other list has lists to hold multiple trees, add one by one
       TIter next(elist->GetLists());
       const TEntryList *e = 0;
       while ((e = (const TEntryList*)next())) {
