--- a/tmva/src/MethodBase.cxx
+++ b/tmva/src/MethodBase.cxx
@@ -2080,7 +2080,7 @@
    // where PDF(x) is the PDF of the classifier's signal or background distribution
 
    if ((reftype == Types::kSignal && !fMVAPdfS) || (reftype == Types::kBackground && !fMVAPdfB)) {
-      Log() << kWARNING << "<GetRarity> Required MVA PDF for Signal or Backgroud does not exist: "
+      Log() << kWARNING << "<GetRarity> Required MVA PDF for Signal or Background does not exist: "
             << "select option \"CreateMVAPdfs\"" << Endl;
       return 0.0;
    }
